apps/codeforafrica/src/components/StayInTouch/Title.js

```diff
@@ -0,0 +1,29 @@
+import { RichTypography } from "@commons-ui/core";
+import { styled } from "@mui/material/styles";
+import PropTypes from "prop-types";
+import React from "react";
+
+const TypographyRoot = styled(RichTypography)(
+  ({ theme: { typography, breakpoints } }) => ({
+    textTransform: "uppercase",
+    fontWeight: "bold",
+    marginBottom: typography.pxToRem(23),
+    [breakpoints.up("md")]: {
+      marginBottom: 0,
+    },
+  })
+);
+
+function Title({ title }) {
```

Comment:
Lets give it a better name and make it a standalone component since it can also be used as title in ShareBar... Something like `StayInTouchTitle`, `ConnectTitle`, `SocialLinksTitle`, etc.